docs/use_cases.md

```diff
@@ -3,4 +3,4 @@ id: use_cases
 title: Use Cases for LDK
 ---
 
-TODO
+LDK is designed to provide incredible flexibility in integrating a lightning node into your application. It is not designed to be a stand-alone application and those seeking a full lightning node to run may wish to look elsewhere. LDK focuses on ensuring tight integration with existing on-chain wallets is easy, allowing reuse of existing blockchain download, key management, UTXO management, and on-disk/cloud storage. See (Getting Started)[getting_started#ldk-batteries] for more details on the interfaces LDK provides for integration.
```

Comment:
Link broken. I don't know how to link within the site so could just make it not a link.